contracts/StrategyPrismaConvexFactoryClonable.sol

```diff
@@ -425,33 +436,54 @@ contract StrategyPrismaConvexFactoryClonable is BaseStrategy {
         returns (address[] memory)
     {}
 
-    function _claimRewards() internal {
-        // By default, we only claim if max boosted. Force claim once if needed.
-        bool _forceClaimOnce = forceClaimOnce;
+    /**
+     * @notice Check if claims via Yearn's locker are max boosted.
+     * @dev This does NOT adjust with set boost delegate address, and will always report status of yPRISMA locker. Thus,
+     *  it is possible this may return true but claims won't be fully boosted depending on delegate selected.
+     * @return Whether claims are max boosted or not.
+     */
+    function claimsAreMaxBoosted() public view returns (bool) {
+        (uint256 claimable, , ) = prismaReceiver.claimableReward(address(this));
+        (uint256 maxBoostable, ) = prismaVault.getClaimableWithBoost(
+            YEARN_LOCKER
+        );
+        return maxBoostable >= claimable;
+    }
 
+    function _claimRewards(
+        bool _forceClaimOnce,
+        address _boostDelegate,
+        uint256 _maxFee
+    ) internal {
+        // By default, we only allow claims if max boosted. Force claim once if needed.
         if (claimsAreMaxBoosted() || _forceClaimOnce) {
             address[] memory rewardContracts = new address[](1);
             rewardContracts[0] = address(prismaReceiver);
             prismaVault.batchClaimRewards(
```

Comment:
Question:

What does `prismaVault.batchClaimRewards` do internally? Only transfer the rewards (ERC20)? Just asking because we usually don't use addresses as input params, and in this case `_boostDelegate` is an external address. Just to confirm it doesn't transfer eth, use callbacks, etc.